ui.schema.json

```diff
@@ -845,260 +845,486 @@
           "properties": {
             // Live Horse Renderer
             "#entity_id": {
-              "type": "string"
+              "oneOf": [
```

Comment:
Whenever it's a string or variable you should either remove the oneOf and put the type as string (recommend since it won't matter. If it doesn't match a var it will still match as a string), or change to "anyOf" instead of "oneOf"